source/public_collections_advanced_search.md

```diff
@@ -0,0 +1,97 @@
+# Public Collections Advanced Search
+
+**_Note that the user-friendliness of advanced searching will improve in the
+coming months._**
+
+### The Default Search Behaviour
+
+When you enter a term into the search bar, the following fields are searched by
+default when no explicit field is given (combined with the `OR` operator):
+`name__icontains`, `owner__username__icontains`,
+`settings__description__icontains`, `summary__icontains`,
+`tags__name__icontains`, `uid__icontains`.
+
+For example, a default search with the term: "_examples_", will result in the
+following:
+
+```
+name__icontains:examples OR owner__username__icontains:examples OR
+settings__description__icontains:examples OR summary__icontains:examples OR
+tags__name__icontains:examples OR uid__icontains:examples
+```
+
+![examples](/images/public_collections_advanced_search/advanced_search_1.png)
+
+### Valid Search Field Operators
+
+The field operator is the value after the last double-underscore in the search
+field, i.e. `__icontains`.
+
+- For _case-sensitive_ **text** searches, the following operators can be used:
+  `contains`, `exact`, `startswith`
+- For _case-insensitive_ **text** searches: `icontains`, `iexact`, `istartswith`
+- For **numeric** search, the following operators are valid: `lte`, `gte`
```

Comment:
`exact` works for numeric fields (really any value), plus there are `lt` and `gt`